pvlib/shading.py

```diff
@@ -232,3 +232,76 @@ def sky_diffuse_passias(masking_angle):
        Available at https://www.nrel.gov/docs/fy18osti/67399.pdf
     """
     return 1 - cosd(masking_angle/2)**2
+
+
+def projected_solar_zenith_angle(axis_tilt, axis_azimuth,
+                                 solar_zenith, solar_azimuth):
+    r"""
+    Calculate projected solar zenith angle in degrees.
+
+    This solar zenith angle is projected onto the plane whose normal vector is
+    defined by ``axis_tilt`` and ``axis_azimuth``. The normal vector is in the
+    direction of ``axis_azimuth`` (clockwise from north) and tilted from
+    horizontal by ``axis_tilt``. See Figure 5 in [1]_:
```

Comment:
Does that mean the axis of tracker rotation ***is*** the surface normal vector we're talking about?

Reply:
From Figure 5 caption:
> AB shows a tracker axis with its tracker rotation plane defined by the 𝒕𝒙 and 𝒕𝒛 axes.

I'm pretty sure it is. Do you think this line of text is worth to be added to the current caption? I didn't want to copy-paste the paper content.

Reply:
How about: 
```suggestion
    This solar zenith angle is projected onto the plane whose normal vector is the torque tube? 
```
 That puts a picture in my mind!

Reply:
That's more focused on tangible parts, I like it.

Nonetheless, I'm a bit hesitant to change that since there is [ongoing discussion](https://github.com/pvlib/pvlib-python/pull/1904#discussion_r1396262134) regarding the tracking or fixed support nomenclature. And that suggests SAT.